sdks/python/apache_beam/options/pipeline_options_validator_test.py

```diff
@@ -93,7 +93,7 @@ def test_missing_required_options(self):
 
     self.assertEqual(
         self.check_errors_for_arguments(
-            errors, ['project', 'staging_location', 'temp_location', 'region']),
+            errors, ['project', 'temp_location', 'staging_location', 'region']),
```

Comment:
I don't understand this change - the order shouldn't matter, right?

Reply:
Rest of the PR LG

Reply:
Fixed this mixup.